package.json

```diff
@@ -29,17 +31,22 @@
     "simple-mock": "^0.8.0",
     "table-builder": "^2.1.1",
     "tap": "^14.0.0",
+    "tslib": "^1.11.1",
```

Comment:
could you explain what you do with `tslib`? I've never used that before

Reply:
> could you explain what you do with `tslib`? I've never used that before

I was playing with `--importHelpers` TypeScript flag (you know, trying to discover a bit of TS), and forgot to remove this dependency after playing with it.